stdlib/src/builtin/object.mojo

```diff
@@ -1178,15 +1105,11 @@ struct object(
             True if the left hand value is greater.
```

Comment:
**Question** Since you changed `eq` and `ne` to return `Bool`, should we have `gt` to return `Bool` too?  Then `object` could model `Comparable` trait, for example.

Reply:
Yep it is a good idea,
would you mind if we leave this open for another day ? 
because it requires doing something like list_compare for tuple
(it is using a `List` too :+1: )
(need to think about how to make list compare be usable by both)

Reply:
Implemented in last commit :+1: ,
but keep in mind that this could not be good, because if the user don't specify an `__eq__` attribute,
we would need a way to raise an error